styles/style.css

```diff
@@ -94,7 +98,17 @@ p {
 
 
 /* Buttons */
-
+button { 
+  color:white;
+  background-color:var(--orange-dark);
+  padding: 0.4rem 0.6rem;
+  border: none;
+  border-radius: 4px;
+}
+button:hover{ 
+  color:var(--orange-dark);
+  background-color: white;
+}
   /* INSERT BUTTON STYLES HERE */
```

Comment:
Should we move button style under this comment? Or remove the comment altogether. Now it is quite confusing for the reader :)

Reply:
Yes